plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/resource/wrapper/LibvirtScaleVmCommandWrapper.java

```diff
@@ -68,6 +71,23 @@ public Answer execute(ScaleVmCommand command, LibvirtComputingResource libvirtCo
         }
     }
 
+    /**
+     * Set the cpu_shares (priority) of the running VM. This is necessary because the priority is only calculated when deploying the VM.
+     * When the number of cores and/or speed of the CPU is changed the cpu_shares is only updated if the VM is restarted or manually, using the command schedinfo.
```

Comment:
I don't understand this sentence; is it updated manually or on restart? Restart seems clear, but can the update be forced manually or is the restart manually?
Also, the sentence below sugests that this is no longer true. I think this needs a re-formulation. Something like "To prevent that the cpu_shares value is only updated in circumstances A and B, this method ....."

Reply:
It can be manually updated through the command mentioned in the PR description: `virsh schedinfo --domain <vm_internal_name> --live cpu_shares=<cpu_shares_value>` or restarting the VM. But I understand it is a bit confusing the way it is written, I will rewrite it.